packages/devextreme/js/__internal/ui/m_tooltip.ts

```diff
@@ -1,96 +1,59 @@
-import $ from '../../core/renderer';
-import Guid from '../../core/guid';
-import registerComponent from '../../core/component_registrator';
-import { extend } from '../../core/utils/extend';
-import Popover from '../popover/ui.popover';
+import registerComponent from '@js/core/component_registrator';
```

Comment:
can we move this file to tooltip folder?

Reply:
I will do it in the next PR to ensure that the history is saved correctly.